internal/contour/listener.go

```diff
@@ -209,6 +285,7 @@ func (l listenersByName) Less(i, j int) bool {
 	return l[i].(*v2.Listener).Name < l[j].(*v2.Listener).Name
 }
 
+// TypeURL returns the cache's Listener Type.
```

Comment:
nit: this is why I don't pay attention to the lint warnings ... this style of comment adds little beyond shutting the linter up.

Reply:
agreed, removed.